api/graphql/schema.graphql

```diff
@@ -628,6 +628,21 @@ input UserPublicProfileFilterInput {
   publicProfileSearch: String @scope
 }
 
+input PoolFilterInput {
+  generalSearch: String @scope
+  name: String @scope
+  ownerName: String @scope
+  ownerEmail: String @scope
```

Comment:
We should probably deprecate these fields on the Pool, actually, and skip adding them here. I don't think we're using them anywhere.

Reply:
Oh yeah, I was wondering if we needed this or not. Removed in [remove owner scopes from pool](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10383/commits/ba6cf3f8d9b866fb1bf7839fb32ab203ac4e8a04)

Just working on the tests now :+1:

Reply:
ownerName and ownerEmail I mean

Reply:
Oh, would you like them to be removed from the `PoolTable` as removing the scopes?